src/Rings/mpoly-localizations.jl

```diff
@@ -1372,9 +1372,9 @@ Ideal generated by
 julia> intersect(I,J,K)
 Ideal generated by
   x*y*w - x*y
-  x^2*w - x^2 + x*y*w - x*y
+  x^2*w - x^2
   x^2*y + x*y^2 + x*y*z
-  x^3 + x^2*y + x^2*z
+  x^3 + x^2*z - x*y^2 - x*y*z
 
 julia> intersect([I,J,K])
 Ideal generated by
```

Comment:
I think the example right below my comment needs to get adapted as well, since it constructs the same intersection as above.